Support/lib/executable.rb

```diff
@@ -56,7 +56,7 @@ class << self
     def find(name, env_var = nil)
       # Safeguard against invalid names so that we don’t need to care about
       # shell escaping later on
-      raise ArgumentError, "Invalid characters found in '#{name}'" unless name =~ /\A[\w_-]+\z/
+      raise ArgumentError.new("Invalid characters found in '#{name}'") unless name =~ /\A[\w_-]+\z/
```

Comment:
Please do not include changes that are not related to the actual issue of the pull request.